tests/terraform/README.md

```diff
@@ -17,6 +17,8 @@ See the [create cluster test](../tests/terraform/createcluster_test.go) as an ex
 Before running the tests, it's required to create a tfvars file in `./tests/terraform/modules/config/local.tfvars`. This should be filled in to match the desired variables, including those relevant for your AWS environment. All variables that are necessary can be seen in [main.tf](../tests/terraform/modules/main.tf).
 It is also required to have standard AWS environment variables present: `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
 
+** Please also when creating tf var resource_name, make sure that you do not have any instances from other automations with the same name to avoid deleting wrong resources
```

Comment:
Can this check be automated as test pre-run?

Reply:
we totally can...but the describe instances command that we should use for that will increase more runtime seconds and this script is already kind of slow because it deletes resources from different places like: load balancers , tgs, r53 and ec2. 

Do you think this tradeoff will worth it ? please let me know you thoughts

Reply:
I was thinking about just failing the test if any resource with that name exists so that the user can just delete them

Reply:
@mdrahman-suse  OH ok , so: 

- For resources that cant be created with the same name,  terraform already do that it breaks the execution.

- For resources like ec2 that can have the same name i dont see it as a good idea fails the test just because of that.  Do you think it is worthy ?

- The warn on the makefile is just about try to avoid create using names that you already commonly use in other places like  i use to create as `fmoral` but in this automation to avoid deletion problems i create like `localMoral` or `fmoral_local` or something else that you use to alias your resources.

Reply:
Good enough, thanks for sharing the details